pom.xml

```diff
@@ -1,30 +1,99 @@
-<project
-        xmlns="http://maven.apache.org/POM/4.0.0"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"
->
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <groupId>org.reactome.release</groupId>
     <artifactId>biomodels</artifactId>
-    <packaging>jar</packaging>
     <version>1.0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
     <name>biomodels</name>
-    <url>http://maven.apache.org</url>
+    <description>
+        Insertion of BioModels identifiers cross-references
+        (data generated by https://github.com/reactome/biomodels-mapper) for Pathway instances
```

Comment:
Is there another tag that denotes dependencies/repositories used, rather then needing to cite in the description in parantheses?

Reply:
I'm not sure I understand what you mean by another tag?

Reply:
It looks like the `parent` and `dependency` tags are used for relationships for other repos, but I don't see any tags that discuss related or dependent repositories not directly used by the source of the current project.